src/EnergyPlus/HeatBalanceSurfaceManager.cc

```diff
@@ -6876,6 +6875,11 @@ void CalcHeatBalanceOutsideSurf(EnergyPlusData &state,
             state, state.dataHeatBalSurf->SurfInsideTempHist(1), 0, state.dataHeatBalSurf->SurfQdotRadNetLWInPerArea, _, Outside);
     }
 
+    // Calculate heat extract due to additional heat flux source term as the surface boundary condition
+    for (int surfNum : state.dataSurface->allOutsideSourceSurfaceList) {
```

Comment:
Unrelated to the main defect here, but streamlined this for a rarely used surface property, same for inside surface source.